tutorials/position-velocity-diagrams/PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,595 @@
+    "im = ax.imshow(pvdiagram3.data)\n",
+    "ax.set_aspect(2.5)\n",
+    "\n",
+    "cb = pl.colorbar(mappable=im)\n",
+    "cb.set_label(\"Brightness Temperature [K]\")\n",
+    "\n",
+    "ax0 = ax.coords[0]\n",
+    "ax0.set_format_unit(u.arcmin)\n",
+    "ax1 = ax.coords[1]\n",
+    "ax1.set_format_unit(u.km/u.s)\n",
+    "\n",
+    "\n",
+    "ax.set_ylabel(\"Velocity [km/s]\")\n",
+    "ax.set_xlabel(\"Offset [arcmin]\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d70082ef",
+   "metadata": {},
+   "source": [
+    "We can also have more widely spaced chunks.\n",
+    "\n",
+    "Note that the spacing given to `extract_pv_slice` affects the shape of the output PV diagram, so we also change the aspect ratio:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "98faa709",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pvdiagram4 = extract_pv_slice(cube=cube, path=skypath2, spacing=5)\n",
+    "\n",
+    "fig = pl.figure(figsize=(12,6))\n",
+    "ax = pl.subplot(121, projection=cube.wcs.celestial)\n",
+    "ax.imshow(mx)\n",
+    "skypath2.show_on_axis(ax, spacing=5,\n",
+    "                      edgecolor='w', linestyle=':',\n",
+    "                      linewidth=0.75)\n",
+    "\n",
+    "ww = wcs.WCS(pvdiagram4.header)\n",
+    "ax.set_xlabel(f\"Right Ascension [{cube.wcs.wcs.radesys}]\")\n",
+    "ax.set_ylabel(f\"Declination [{cube.wcs.wcs.radesys}]\")\n",
+    "\n",
+    "ax = pl.subplot(122, projection=ww)\n",
+    "im = ax.imshow(pvdiagram4.data)\n",
+    "cb = pl.colorbar(mappable=im)\n",
+    "cb.set_label(\"Brightness Temperature [K]\")\n",
+    "ax.set_aspect(0.5)\n",
+    "\n",
+    "ax0 = ax.coords[0]\n",
+    "ax0.set_format_unit(u.arcmin)\n",
+    "ax1 = ax.coords[1]\n",
+    "ax1.set_format_unit(u.km/u.s)\n",
+    "\n",
+    "ax.set_ylabel(\"Velocity [km/s]\")\n",
+    "ax.set_xlabel(\"Offset [arcmin]\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c8203c9d",
+   "metadata": {},
+   "source": [
+    "## Saving\n",
+    "\n",
+    "Finally, we can save the extracted PV diagram as a FITS file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d59d2ff3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pvdiagram.writeto(\"saved_pvdiagram.fits\", overwrite=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "560c2f93-ee95-4d60-825a-83ec9f3acba7",
+   "metadata": {},
+   "source": [
+    "We can also save the figure as a png or pdf:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74de9f99-e2fd-411f-9249-a39fa0b35908",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig.savefig(\"saved_pvdiagram.png\", bbox_inches='tight')\n",
+    "fig.savefig(\"saved_pvdiagram.pdf\", bbox_inches='tight')"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Py 3.9",
+   "language": "python",
+   "name": "python39"
```

Comment:
```suggestion
   "name": "python3"
```

Would you mind making this single change, directly on the JSON (without saving it against via the Jupyter app?)